deploy/crds/spinnaker_v1alpha1_spinnakerservice_crd.yaml

```diff
@@ -59,10 +59,16 @@ spec:
                             additionalProperties:
                               type: string
                             type: object
+                          port:
```

Comment:
what do you think about adding clarity on this by changing `port` to `exposedPort` or maybe `publicPort` or.... hmmmmmmmmm....... I guess `port` is ok?

Reply:
`publicPort` sounds good